docs/i18n/en/docusaurus-plugin-content-docs/version-3.0.0.json

```diff
@@ -0,0 +1,10 @@
+{
+  "version.label": {
+    "message": "3.0.0",
+    "description": "The label for version 3.0.0"
+  },
+  "sidebar.docs.category.Introduction": {
+    "message": "Introduction",
+    "description": "The label for category Introduction in sidebar docs"
```

Comment:
```suggestion
    "description": "The label for category introduction in sidebar docs"
```

Reply:
This is produced when producing translations file. `Introduction` is the category name in the description. Also, it is auto-generated.